content/millennium/dstu2/diagnostic/observation.md

```diff
@@ -44,14 +45,19 @@ _Implementation Notes_
 
 * Currently, the `comments` field may have RTF or other formatted data rather than simple text. This is an issue that will be resolved in a future correction.
 
+* When multiple pages of Observation results are returned for a single query:
+  * All Social history Observations (if any qualify for the query) will be returned on the first page of results. This means that the next bullet does not apply to Social history Observations.
+  * Results are sorted by effective date/time in descending order by page. That is, all Observations on the any given page of results are newer than all Observations on the next page of results. Sort order within pages is not guaranteed.
```

Comment:
Phrasing seems odd "all Observations on the any given page"

Reply:
oops, didn't delete 'the' when I was rephrasing that sentence
https://github.com/cerner/fhir.cerner.com/pull/270/commits/b9c10f3594b4e20048d36134dc14083ec32cc1eb